geode-redis/src/integrationTest/java/org/apache/geode/redis/SSLTest.java

```diff
@@ -41,8 +41,8 @@
   @ClassRule
   public static ServerStarterRule sslEnabledServer = new ServerStarterRule()
       .withSSL("server", true, true)
-      .withProperty("redis-bind-address", "localhost")
-      .withProperty("redis-port", "11211")
+      .withProperty("compatible-with-redis-bind-address", "localhost")
+      .withProperty("compatible-with-redis-port", "11211")
       .withProperty("redis-enabled", "true")
```

Comment:
Is the `redis-enabled` property ok or should that be changed as well?

Reply:
I made a whole bunch of changes, including the ones you mentioned. thank you for reviewing so quickly though :)